### PR TITLE
For commenting on 01-04-Nonuniform-sensitivity.ipynb (part of review-865a5c2)

### PR DESCRIPTION
**Pull request by mwcraig**

### CODE REVIEW

File: notebooks/01-04-Nonuniform-sensitivity.ipynb [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+   "source": [
+    "## Background\n",
+    "\n",
+    "Not all pixels in a camera have the same sensitivity to light: there are\n",
+    "intrinsic differences from pixel-to-pixel. Vignetting, a dimming near the\n",
+    "corners of an image caused by the optical system to which the camera is\n",
+    "attached, and dust on optical elements such as filters, the glass window\n",
+    "covering the CCD, and the CCD chip itself can also block some light.\n",
+    "\n",
+    "Vingetting and dust can reduce the amount of light reaching the CCD chip while\n",
```
Comment (saimn) [inline notebooks/01-04-Nonuniform-sensitivity.ipynb]:
```suggestion
    "Vignetting and dust can reduce the amount of light reaching the CCD chip while\n",
```

File: notebooks/01-04-Nonuniform-sensitivity.ipynb [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    "pixel-to-pixel sensitivity variations affects the counts read from the chip.\n",
+    "\n",
+    "The code to produce the simulated sensitivity map (aka flat image) is long\n",
+    "enough that is not included in this notebook. We load it instead from\n",
+    "[image_sim.py](image_sim.py)."
```
Comment (saimn) [inline notebooks/01-04-Nonuniform-sensitivity.ipynb]: File not found.

Comment (mwcraig) [reply notebooks/01-04-Nonuniform-sensitivity.ipynb:27]: Thanks, checking on this

**Comment by mwcraig** [comment]
Fixed in #233

**Pull request closed without merging**